test/e2e/scheduling/preemption.go

```diff
@@ -467,6 +468,71 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 			}
 		}
 	})
+	// this test makes sure that Pods with a higher PriorityClass via their PriorityClassName start earlier than Pods with a lower PriorityClassName
+	ginkgo.It("should ensure Pods using a PriorityClassName which is higher begin first", func() {
+		podNamesSeen := []string{}
+		stopCh := make(chan struct{})
+		pods := []v1.Pod{}
+		// define Pod specs - order in from highest to lowest priority
+		ginkgo.By("Defining Pod specs")
+		for i := 4; i >= 1; i-- {
+			pods = append(pods, v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: fmt.Sprintf("priority-class-p%v", i),
+				},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{{
+						Name:    "priority-class",
+						Image:   imageutils.GetE2EImage(imageutils.Agnhost),
```

Comment:
Why using this image?

Reply:
I chose this image because it's platform agnostic

Reply:
These are the defaults now and the correct images to use.